src/Services/TransactionService.php

```diff
@@ -74,4 +74,20 @@ public function refund($transactionId, $amount)
             ['transactionId' => $transactionId]
         );
     }
+
+    /**
+     * @param array|JsonSerializable|Transaction $data
+     * @param float $amount
+     *
+     * @throws UnprocessableEntityException The input data does not valid
+     *
+     * @return Transaction
+     */
+    public function create($data)
```

Comment:
where is amount? or is the docblock wrong?

Reply:
sorry. docblock is wrong. will fix it